index.html

```diff
@@ -3674,11 +3709,15 @@ <h3>HTML Attribute State and Property Mappings</h3>
               </td>
               <td class="ax">
                 <span class="type">Role:</span>
-                <a href="#el-textarea">AXtextArea</a>
+                <a href="#el-textarea">AXTextArea</a>
                 <div class="general">Use WAI-ARIA mapping</div>
               </td>
               <td class="comments">
-                If the element has the `contenteditable` attribute  and `aria-readonly="true"`, User Agents MUST expose only the `contenteditable` state.
+                <p>If the element has the `contenteditable` attribute  and `aria-readonly="true"`, User Agents MUST expose only the `contenteditable` state.</p>
```

Comment:
```suggestion
                <p>If the element has the `contenteditable` attribute and `aria-readonly="true"`, User Agents MUST expose only the `contenteditable` state.</p>
```

I apologize for this objectively useless suggestion.